src/se/vidstige/jadb/Stream.java

```diff
@@ -20,4 +20,11 @@ public static String readAll(InputStream input, Charset charset) throws IOExcept
         Stream.copy(input, tmp);
         return new String(tmp.toByteArray(), charset);
     }
+
+    public static void flushRead(InputStream is) throws IOException {
+        int v;
+        do{
```

Comment:
Single space between `do` and `{`. Also code is duplicated in `JadbDevice.java`. Never copy-paste code.